src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -446,6 +447,16 @@ export class DynamicFacet extends Component implements IDynamicFacet {
     this.updateQueryStateModel();
   }
 
+  /**
+   * Keyboard focuses a value once it has been re-rendered.
+   *
+   * @param value The name of the facet value to focus
+   */
+  public focusValueAfterRerender(value: string) {
```

Comment:
Good point. Unfortunately, the alternatives aren't very pleasant either. We could try to expose an event and retrieve the new facet value from the old one, which seems pretty hack-ish too. I think I overall prefer exposing a somewhat useless function, especially since JSUI is mostly just in maintenance.